powerlevel9k.zsh-theme

```diff
@@ -189,7 +189,7 @@ left_prompt_segment() {
 # End the left prompt, closes the final segment.
 left_prompt_end() {
   if [[ -n $CURRENT_BG ]]; then
-    echo -n "%k%F{$CURRENT_BG}$(print_icon 'LEFT_SEGMENT_SEPARATOR')"
+    echo -n "%k$(foregroundColor ${CURRENT_BG})$(print_icon 'LEFT_SEGMENT_SEPARATOR')"
```

Comment:
`$CURRENT_BG` will contain a numeric, since `getColorCode` was already called in `left_prompt_segment`. Do we really have to add another function call to slow the prompt down more?

Reply:
Yes, `$CURRENT_BG` is already numeric. I made this change anyway, to match up with the other occurrences of foreground color. To go full nuts I removed the possibility to stop an fore/background color with a call to that method without params.

In an upcoming release we should either go all in with helper methods for getting a prepared string that contains "start a foreground color", or nothing. I am undecided between the two ways. In another programming language I would prefer a helper method that wraps a cryptic DSL to a nice interface, but here I would indeed prefer going for speed and not call a subshell for that. We should think about a concept for that.